.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: c
 sudo: false
+dist: focal
```

Comment:
What does this do?

Reply:
This changes the Ubuntu distribution used for Travis, which was one of the suggestions in ocaml/ocaml-ci-scripts#351 for handling the build error (see the failed build log on the first commit).